### DISCUSSION TITLE
FNIRSI GC-01: M4011 tube

### DISCUSSION THREAD

**Discussion by jasperweiss** [Gissio/radpro#127] [Q&A] [answered]
Hi everyone, I just got a GC-01 from the official F-NIRSI store on AliExpress and I was wondering if anyone could confirm this is a M4011 tube? The tube has GM4011H24.05 written on it. Just wondering if that's a "real" M4011. If so, how good are these compared to J321's?

Most images on internet of M4011 tubes just have "M4011" written on it along with the year it was produced

![signal-2024-07-18-214907_002](https://github.com/user-attachments/assets/268bb8a0-f2e3-46d5-a994-26fe73bd78b6)

**Comment by jonwhite227** [comment, marked as answer]
That's a new M4011, the date is May 2024 (24.05), so fairly recent production. The M4011 tube is virtually identical to the J321 - one is a couple of mm longer than the other - can't remember which. Basically it's just as good as the J321, I've used both with the same results.

**Reply by jasperweiss** [reply]
Thanks! So far it seems to work well with radpro. I was just curious about the "GM" and "H" as opposed to just "M" in the model name

**Comment by naviricbe** [comment]
I have same tube inside my GC-01 but there is small hole on the metal end of the tube, I don't know if that is normal or not ?

**Reply by jonwhite227** [reply]
Yes, it's normal. 👍

**Comment by ThatBum** [comment]
My device's tube literally has "J321βγ (M4011) XRHH 2024 IV" on it, so it's pretty safe to say they're interchangeable. 😄 I think it's just different nomenclature/model numbers between manufacturers. Dunno about the XRHH though... something to do with x-rays?

![PXL_20250207_193859537](https://github.com/user-attachments/assets/6be5c079-dae2-4db5-8ae1-5726c30ed861)
